test/mpi/pt2pt/testlist.in

```diff
@@ -24,9 +24,7 @@ rqstatus 2
 rqfreeb 4
 greq1 1
 probe_unexp 4
-probe_unexp 4 env=MPIR_CVAR_CH4_EAGER_MAX_MSG_SIZE=16384
 probenull 1
-probenull 1 env=MPIR_CVAR_CH4_EAGER_MAX_MSG_SIZE=16384
```

Comment:
One of the reasons we added this was because some OFI providers set their "eager" limit to be something huge (e.g. 2GB) and handle eager vs. rendezvous internally. Having this CVAR allowed us to tweak this value externally, specifically for testing.

Is there a replacement for that? If so, can we replace these tests with those values?

Reply:
I am not sure if that the case. The OFI code today have a provider-independent eager limit https://github.com/pmodels/mpich/blob/77199f67bfe2c3663e6c37476c9866479a6ef68c/src/mpid/ch4/netmod/ofi/ofi_am.h#L125

Reply:
After some investigation. Now I see why I couldn't understand the purpose of this CVAR.

I found this CVAR was used to set `MPIDI_OFI_global.max_msg_size` which controls is the eager limit on the native path and not the AM path.

For some reason this was changed to only affect AM path. As a result OFI no longer sets the `MPIDI_OFI_global.max_msg_size` to this CVAR. We must have been testing for months with OFI native path using whatever provider says as eager limit 😱 .

Reply:
I will work on a fix.

Reply:
I think in principle, we should allow user an option to override EAGER threshold -- up to certain internal limit.